rouzzz/App.js

```diff
@@ -95,9 +141,25 @@ export class CountdownScreen extends React.Component {
           title="Go to Alarm"
           onPress={() => this.props.navigation.navigate('Alarm')}
         />
+        <Button
+          title="Go to TimeFinder (Debug page)"
+          onPress={() => this.props.navigation.navigate('TimeFinder')}
+        />
       </View>
     );
   }
+  componentDidMount() {
+    setInterval(() => {
+        this.setState(() => {
+            if(Date.now() >= this.date.getTime()){
+              this.props.navigation.navigate('Alarm')
+            }
+            return { unseen: "does not display" }
+        });
+    }, 1000);
```

Comment:
It's more of a pain for testing, but in reality, waking up 1 minute or so later really doesn't mean anything. You should change it to something like 1 minute+ in master, but you can shorten it when you test locally.